domains/xcody.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "theinfamouscoder5",
+        "email": "codingamazing224@outlook.com"
+    },
+    "record": {
+        "CNAME": "https://hi-im-cody.pages.dev"
```

CNAME can't contain protocol
```suggestion
        "CNAME": "hi-im-cody.pages.dev"
```